libs/blocks/video/video.css

```diff
@@ -1,3 +1,7 @@
+a[href*='.mp4'].hide {
+  visibility: hidden !important;
```

Comment:
`!important` should not be necessary.

Reply:
This is for same pattern of hiding other auto blocks:
https://github.com/adobecom/milo/blob/main/libs/blocks/caas/caas.css
https://github.com/adobecom/milo/blob/main/libs/blocks/faas/faas.css
This is little more general since this is no parent class for the a link and I think it is safe to have !important specially with `.hide`. (like bootstrap does.).
If you still disagree having !important for this case,  I can remove it.

Reply:
strange that without `!important` it doesn't work, anyway not a blocker for me.